example/lambdas/python-reference-task/Pipfile

```diff
@@ -13,4 +13,4 @@ six = "==1.12.0"
 pylint = ">=1.8.2"
 
 [requires]
-python_version = "3.7"
+python_version = "3"
```

Comment:
Willing to revert these if needed, but it fixes endless annoyance on my machine.

Reply:
Can you elaborate?     I'm pretty sure we want to explicitly state the python subversion as we're using idioms that aren't compatible with < 3.7, and if we *aren't*, we almost certainly will be.

Reply:
It won't allow python >3.7 either, in my case 3.8.9.

Reply:
Choosing not to die on this hill.